src/features/ITGovernance/Admin/Dates/index.tsx

```diff
@@ -1,277 +1,382 @@
+      ).toISO();
+
+      await updateReviewDates({
+        variables: {
+          input: {
+            id: systemId,
+            grtDate: newGrtDate,
+            grbDate: newGrbDate
+          }
+        }
+      });
+
+      history.push(`/it-governance/${systemId}/intake-request`);
+    }
   };
 
+  /**
+   * Returns the correct status data for the review type,
+   * or NOT_STARTED if status is null
+   */
+  const grbReviewStatus: GRBReviewStatus | null | undefined = useMemo(() => {
+    if (!grbReviewStartedAt) return 'NOT_STARTED';
+
+    return systemIntake.grbReviewType === SystemIntakeGRBReviewType.STANDARD
+      ? grbReviewStandardStatus
+      : grbReviewAsyncStatus;
+  }, [
+    grbReviewAsyncStatus,
+    grbReviewStandardStatus,
+    grbReviewStartedAt,
+    systemIntake.grbReviewType
+  ]);
+
+  const reviewIsInProgress: boolean =
+    grbReviewStatus !== 'NOT_STARTED' && grbReviewStatus !== 'COMPLETED';
+
+  if (!grbReviewStatus) {
+    return null;
+  }
+
   return (
-    <Formik
-      initialValues={initialValues}
-      onSubmit={onSubmit}
-      validationSchema={DateValidationSchema}
-      validateOnBlur={false}
-      validateOnChange={false}
-      validateOnMount={false}
-    >
-      {(formikProps: FormikProps<SubmitDatesForm>) => {
-        const { errors, setErrors, handleSubmit } = formikProps;
-        const flatErrors = flattenErrors(errors);
-        return (
-          <>
-            {Object.keys(errors).length > 0 && (
-              <ErrorAlert
-                testId="system-intake-errors"
-                classNames="margin-top-3"
-                heading={t('form:inputError.checkFix')}
-              >
-                {Object.keys(flatErrors).map(key => {
-                  return (
-                    <ErrorAlertMessage
-                      key={`Error.${key}`}
-                      errorKey={key}
-                      message={flatErrors[key]}
+    <>
+      <PageHeading data-testid="grt-dates-view" className="margin-top-0">
+        {t('governanceReviewTeam:dates.heading')}
+      </PageHeading>
+      <h2>{t('governanceReviewTeam:dates.subheading')}</h2>
+      <Form onSubmit={handleSubmit(onSubmit)} className="maxw-mobile-lg">
+        <fieldset className="usa-fieldset margin-top-4">
+          <legend className="usa-label margin-bottom-1">
+            {t('governanceReviewTeam:dates.grtDate.label')}
+          </legend>
+          <HelpText id="TestDate-DateHelp-grt">
+            {t('governanceReviewTeam:dates.grtDate.format')}
+          </HelpText>
+          <div className="usa-memorable-date">
+            <div className="usa-form-group usa-form-group--month">
+              <Controller
+                name="grtDateMonth"
+                control={control}
+                render={({
+                  field: { ref, ...rest },
+                  fieldState: { error }
+                }) => (
+                  <FormGroup className="margin-top-2" error={!!error}>
```

Comment:
The `Controller` components are already wrapped in a div with the class `usa-form-group` for styling purposes. I don't think there needs to be another div with the same class here.

Also, adding the error state to this `FormGroup` is part of why the error states are showing up differently.